model/src/main/proto/oppia_logger.proto

```diff
@@ -368,6 +372,30 @@ message EventLog {
     string profile_id = 2;
   }
 
+  // Structure of a feature flag context.
+  message FeatureFlagContext {
+    // The unique UUID of the user.
+    string uuid = 1;
```

Comment:
Has this been implemented yet? I have not seen it being set in the logger.

Reply:
No. We don't have a UUID yet. I was future-proofing the code so that in case we create one we just plug and play.

Reply:
I think it could benefit from a more descriptive name and comment.

Reply:
I renamed this to unique_user_uuid. I'm not sure if that is what you had in mind. Let me know if I need to improve it further.